yesod-test/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 1.5.7
+
+* Add clickOn
```

Comment:
Could you link to this PR in the changelog so people can get further details? Just line for 1.5.4